### DISCUSSION TITLE
Helm Chart: PostgresSQL not redundant but has redundant redis?

### DISCUSSION THREAD

**Discussion by voarsh2** [chatwoot/chatwoot#4553] [Q&A] [answered]
I'm running Chatwoot on K8 using the official Helm chart and am quite surprised that PostgreSQL is running as a Statefulset that is not in a HA fashion.

I cannot increase the Statefulset count to more than 1 as it is not seemingly setup for high availability. 

But... the Helm Chart has HA Reddis? Am I missing anything?

**Comment by vishnu-narayanan** [comment, marked as answer]
@voarsh2 No, you are right. Just that we started off with it. We are using the Redis/Postgres charts packaged by Bitnami as sub-dependency to the main chart. Planning to change it to HA Postgres deployment.
